src/modules/auto-tracked-scroll.js

```diff
@@ -24,15 +24,20 @@
 		scrollH = bodyH - winH;
 	};
 	
-	var onEnter = function (next, data) {
+	var onEnter = function (key, data) {
+		var root = !!data.page ? $(data.page.key()) : data.article ? $(data.article) : $();
 		tracker.init();
-		App.callback(next);
 		setTimeout(onResize, 100);
+		if (root.hasClass('js-no-tracked-scroll')) {
```

Comment:
Maybe `js-no-tracked-scroll` could be a constant ?

Reply:
I Suggest also renaming to js-disable-tracked-scroll or js-tracked-scroll-disabled.

Reply:
Both fixed in 169ac30